app/controllers/reviews_controller.rb

```diff
@@ -1,17 +1,22 @@
 class ReviewsController < ApplicationController
-  before_action :find_restaurant, only: [:new, :create]
+  before_action :find_restaurant, only: [:create]
+
+  def show
+    @review = Review.find(params[:id])
+  end
 
   def new
     @review = Review.new
   end
 
   def create
-    @review = Review.new(review_params)
-    @review.restaurant = @restaurant
+    @review = @restaurant.reviews.new(user: current_user)
+
+    binding.pry
     if @review.save
-      redirect_to restaurant_path(@restaurant)
+      redirect_to restaurant_review_path(@restaurant, @review)
```

Comment:
shouldn't line 17 redirect to the restaurant after saving?

successful save: back to Restaurant => restaurant_path(@restaurant)

Reply:
@georg - we have nested ingredient reviews inside reviews (ingredient reviews have a review_id).  This "link" file (reviews table) means we have three levels: restaurant => reviews => ingredient_reviews.

We need to create a "review object" so we can create an ingredient review.  We need a review_id which comes from the review that we just created to add to the ingredient review.  

From what I researched, this is complicated and requires either nested forms OR a review being created first and adding ingredient reviews to that newly-created review object.

That's why I redirected to that review "show" page.  That will have the page to add multiple ingredient_reviews.